setup.py

```diff
@@ -31,11 +31,14 @@
     )
 ]
 
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
 setup(
     name="editdistance",
     version='0.5.3',
     description="Fast implementation of the edit distance(Levenshtein distance)",
-    long_description='',
+    long_description=readme,
```

Comment:
```suggestion
    long_description=readme,
    long_description_content_type ='text/x-rst',
```
Since [PyPi supports multiple styles](https://packaging.python.org/guides/making-a-pypi-friendly-readme/) it would be good to specify that the `long_description` is in rst. 😄